### DISCUSSION TITLE
support react18

### DISCUSSION THREAD

**Discussion by yuhang1995** [vechai/vechaiui#75] [General]
Hello, I just discovered this precious library today, I love it, it's really cool, when will react18 be supported

**Comment by pepelele** [comment]
sorry guys but I'm very busy right now.I have to work all days of the week...so I haven't had time to work on vechai v1 or upgrade to react 18. I wish I could take a break to improve vechai 😭

**Reply by yuhang1995** [reply]
It's too hard, but thank you for your hard work, and you still need to rest properly.
